<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Try Catch statment in Error Handling</title>
</head>
<body>
    <form action="">
        <h3>Enter two number you want to divide it</h3>
        Enter number-1: <input type="text" id="number-1">
        Enter number-1: <input type="text" id="number-2">

        <button onclick="divideFunction()">Calculate</button>
    </form>
     <!-- <button onclick="divide()">click</button> -->
    <script>
        function divide(a,b){
            try{
                var c;
                // var a=12;
                // var b=0;
                c=parseInt(a/b);

                document.write(c);
            }
            catch(e){
                console.log("Error: "+ e.message);
            }
            finally{
                document.write("finally colsed");
            }
        }

        function divideFunction(){
            var a=parseInt(document.getElementById("number-1").value);
            var b=parseInt(document.getElementById("number-2").value);
            divide(a,b);
        }
    </script>
</body>
</html>